panel/browser-panel.cpp

```diff
@@ -280,24 +281,42 @@ void QCefWidgetInternal::Resize()
 	QueueCEFTask([this, size]() {
 		if (!cefBrowser)
 			return;
+
+#ifdef _WIN32
 		HWND hwnd = cefBrowser->GetHost()->GetWindowHandle();
 		SetWindowPos(hwnd, nullptr, 0, 0, size.width(), size.height(),
 			     SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER);
 		SendMessage(hwnd, WM_SIZE, 0,
 			    MAKELPARAM(size.width(), size.height()));
-	});
+#elif __APPLE__
+#else
+		auto browser_win = cefBrowser->GetHost()->GetWindowHandle();
```

Comment:
I think in C++ files we use camelCase for variables.  I'd probably use `browserWindow` for this.